### DISCUSSION TITLE
Reimport von Titelmetadaten einzelner Vorgänge

### DISCUSSION THREAD

**Discussion by andre-hohmann** [kitodo/kitodo-production#5790] [Ideas]
# Einleitung

Im [Community Board (Kitodo.Production)](https://github.com/kitodo/kitodo-production/wiki/Community-Board-(Kitodo.Production)) wird die Implementierung eines Reimports diskutiert ([2023‐08‐08](https://github.com/kitodo/kitodo-production/wiki/2023%E2%80%9008%E2%80%9008-Community-Board-(Kitodo.Production)), [2023‐09‐19](https://github.com/kitodo/kitodo-production/wiki/2023%E2%80%9009%E2%80%9019-Community-Board-(Kitodo.Production))). Auslöser ist die Diskussion während des Kitodo-Praxistreffens 2022 und Issue #5410 sowie PR #5613. 

Ziel ist zunächst die Erarbeitung von Vorschlägen zur Umsetzung der Funktionalität für einzelne Vorgänge im Metadateneditor. Unter anderem werden folgende Aspekte untersucht: 

* Position des Buttons, um den Reimport auszulösen
* Informationen über Änderung der Daten

Darauf aufbauend werden perspektivisch Vorschläge zur Massen- oder Stapelbearbeitungen größerer Mengen erarbeitet. Zum Beispiel ob die KitodoSkripte erweitert werden können oder ob ein neues Konzept entwickelt wird. Zudem muss noch unter anderem geklärt werden, wie der korrekte Katalog oder die korrekte Metadatenquelle ausgewählt wird. 

In dieser GitHub-Diskussion werden die bisherigen Erkenntnisse des Community Board zusammengeführt, um eine Diskussion in der gesamten Community anzuregen. Gleichzeitig sollen die Herausforderungen einer nachhaltigen Lösung hinsichtlich aller Aspekte (technisch, fachlich, Nutzerfreundlichkeit ...) vermittelt werden. 

# Position des Buttons, um den Reimport auszulösen
Im Community Board (Kitodo.Production) wurden zwei Lösungsansätze diskutiert: 

1. Button im „Kopf“ des Frame Metadaten
2. Button im "Kopf" des Metadateneditors

Lösungsansatz 1 wird derzeitig im Community Board favorisiert. 

## Button im „Kopf“ des Frame Metadaten
Lage: Entsprechend der Funktionalitäten im "Kopf" des Strukturbaums (Ein- und Ausklappen des Strukturbaums, ...) 

**Vorteile**
* Ähnliche Struktur wie im Frame „Strukturdaten“
* Funktionalität entspricht der Metadatenbearbeitung
* Der „Kopf“ des Metadateneditors bleibt übersichtlicher

**Nachteile**
* Der Frame Metadaten wird etwas kleiner
* Die Position ist nicht so prägnant wie im „Kopf“ des Metadateneditors

**Voraussetzungen**
* Mouseover mit Erläuterung der Funktion (Vergleiche Buttons Einklappen/Ausklappen des Strukturbaums) 
* Prüfen, ob der +-Button zum Hinzufügen von Metadaten ebenfalls in den Kopf des Frames eingefügt wird
* Vermittlung, dass sich die Funktion auf die Katalogmetadaten bezieht, nicht auf die manuell erfassten Metadaten der Strukturdaten

**Beispiel**
![MockupReimportButtonStruktur01](https://github.com/kitodo/kitodo-production/assets/14835770/ad575e9d-5d47-4241-b54c-c3198d4ec7e8)

## Button im „Kopf“ des Metadateneditors
Lage: Neben „Schließen“, „Validieren“, …

**Vorteile**
* Prägnante Position, die rasch erkannt wird

**Nachteile**
* Die Zeile wird rasch voll, wenn andere Aktionen angezeigt (Bildbearbeitung, …) oder in Zukunft hinzugefügt werden
* Inhaltlich entspricht es weniger den eher allgemeinen Aktionen Speichern, Zurück, …

**Beispiel**
![MockupReimportButtonEditor01](https://github.com/kitodo/kitodo-production/assets/14835770/c9082157-d8dd-4c15-874b-d019357184d3)

# Informationen über Änderung der Daten

**Funktionalität**
Bevor die Metadaten aktualisiert werden, sollen folgende Funktionen angeboten werden:

* Anzeige der Änderungen, die durchgeführt werden
* Möglichkeit, die Art der Anpassung (replace, add, …) anzupassen
* Anwendung des Reimports auf alle oder nur ausgewählte Metadatenfelder 

**Beispiel**
![MockupReimportAnzeige01](https://github.com/kitodo/kitodo-production/assets/14835770/6cf4de13-e279-4857-85b1-4e3d739c5752)

**Comment by markusweigelt** [comment]
Mein erster Eindruck davon ist, dass die zwei Lösungsansätze optisch verschiedene Funktionalitäten vermitteln und ich mir auch gut vorstellen kann, dass beide Funktionen Einsatz finden.

Der Button im oberen Bereich des Metadateneditors suggeriert mir, dass es sich um einen Reimport aller Metadaten aus allen Strukturelementen handelt bzw. der Nutzer aus allen Metadaten auswählen kann. Der Button innerhalb des Frames Metadaten bezieht sich nach meinem Eindruck nur auf die Metadaten, die sich innerhalb des Frames befinden.

Je nach Anwendungsfall kann es sinnvoll sein eine komplette Liste der Metadaten für Reimport durchzugehen oder nur die dem Strukturelement zugeordnet sind. Beide Buttons müssen sich meiner Meinung nach aber nicht ausschließen. Im Backend wäre es sicher dieselbe Funktionalität. Der Unterschied wäre lediglich, dass beim Frame Metadaten auf die Metadaten des ausgewählten Strukturelementes gefiltert wird..

Mit dem bereits gemergten PR https://github.com/kitodo/kitodo-production/pull/5701 zur Mediendateiumbenennung, gibt es bereits einen weiteren Button ab Release 3.6 der sich im oberen Bereich befinden kann (je nach Berechtigungen). Vielleicht könnte im "Kopf" des Metadateneditors ähnlich der Vorgangsübersicht ein Button "Aktionen" hinzugefügt werden und der Reimport aus den Aktionen ausgewählt werden. Der Button für die Mediendateiumbenennung könnte auch in die Aktionen verschoben werden.

Zusammenfassend kann ich mir beide Buttons vorstellen bzw. den Button je nach präferierten Anwendungsfall.

**Comment by oliver-stoehr** [comment]
Ich hatte vor längerer Zeit einen Wireframe für das Dialogfenster der Metadaten-Aktualisierung erstellt. Ich habe dies nun in ein Mockup überführt und will es zu dieser Diskussion beisteuern:
<img width="919" alt="Bildschirmfoto 2023-09-29 um 13 18 30" src="https://github.com/kitodo/kitodo-production/assets/32509444/a8b7adcf-5f97-406e-ac7a-e32a81ea5445">
In diesem Konzept sollten Record-Identifier und Katalog in der Überschrift stehen und rechts ein kleines Hinweis-Icon, über das ein Erklärungstext zu dem Dialogfenster eingeblendet werden kann (ähnlich den schönen Hinweisen bei den Importkonfigurationen).

Die eigentliche Tabelle soll links die Metadaten-Keys, in der zweiten Spalte die bisherigen Werte aus Kitodo und in der letzten Spalte die neuen, aus dem Katalog geladenen Werte auflisten.  
Die dritte Spalte soll möglichst intuitive Buttons zur Steuerung beinhalten. Diese Buttons haben jeweils drei Zustände, wo der Pfeil jeweils auf den Wert zeigt, der übernommen werden soll:
- `-->` (ersetzen)
- `<--` (behalten)
- `<-->` (ergänzen)

In diesem Mockup würde also der bisherige Titel beibehalten werden, die bisherige Sprache um die neue, aktualisierte Sprache ergänzt werden, das bisherige Erscheinungsjahr beibehalten werden und der Erscheinungsort ersetzt werden.

Durch einfaches Klicken auf einen Button soll durch die Zustände durchgeschaltet werden können. Wäre der Zustand `-->` "ersetzen" ausgewählt würde der erste Klick zu `<--` "behalten" wechseln, der zweite Klick zu `<-->` "ergänzen".
Dadurch wäre für viele Änderungen nur ein Klick notwendig und der aktuelle Zustand wäre schnell und ohne texthelle Erläuterungen zu erkennen.  
Ein Klick auf das Symbol in der Titelzeile würde den Zustand für alle Zeilen auf `-->` "ersetzen" ändern, ein zweiter Klick würde alle Zeilen auf `<--` "behalten" ändern und ein dritter Klick auf `<-->` "ergänzen". So könnten mit wenigen Klicks alle Zeilen angepasst werden. Einzelne Zeilen könnten danach abgeändert werden, wenn ein Update zum Beispiel viele Metadaten lädt, aber nur einzelne Zeilen davon übernommen werden sollten.

Zwecks besserer Eindeutigkeit sollten die Metadaten, die durch die entsprechenden Auswahlen übernommen werden würden, hervorgehoben werden. In diesem Konzept sind die entsprechenden Einträge **fett** geschrieben, eine andere Art der Hervorhebung wäre aber auch denkbar (Rahmen um die Wert, Textfarbe, Hintergrundfarbe oder Ähnliches).

**Reply by BartChris** [reply]
Nach Diskussionen hierzu im @kitodo/kitodo-community-board ist eine eine der offenen Fragen, woher die für den Reimport notwendige Information über das Metadaten-Quellsystem (=die Importkonfiguration) und den für die Abfrage des Katalogs notwendigen Identifier stammen.

Damit der Re-Import funktioniert muss diese Information irgendwo in Kitodo hinterlegt sein. Hierfür kommt a) die Datenbank oder b) die METS-Datei in Frage.
Bei Nutzung der METS-Datei böte sich zur Speicherung des Katalogschlüssels im Metadatensystem die Auswertung des Felds mit dem funktionalen Attribut `use=recordIdentifier` an. In der Praxis ist es allerdings so, dass in diesem Feld nicht immer der Katalogschlüssel steht, wie er zur Abfrage des Katalogsystems benötigt wird. So wird mitunter im als `recordIdentifier` markierten Feld ein Präfix PPN vorangestellt. Das Feld ist daher nur bedingt geeignet um (ohne manuelle Anpassung) für den Reimport der Metadaten ausgewertet zu werden.
Da generell Annahmen über die genaue Belegung von Feldern im Ruleset schwierig sind, böte sich alternativ an, die Information über die verwandte Importkonfiguration und den eingesetzten Katalogschlüssel in der Kitodo-Datenbank zu speichern. Die verwandte Import-Konfiguration und der für den Import eingesetzte Katalogschlüssel könnten direkt in dem Datenbank-Eintrag des Vorgangs hinterlegt und genutzt werden. Bei der Nutzung der Datenbank wäre vermutlich ein Instrument zu finden, wie bei Anlage des Vorgangs sichergestellt ist, dass die entsprechende Information auch in der Datenbank hinterlegt ist. Da Kitodo-Vorgänge auch ohne Kataloganbindung erzeugt werden können, muss dementsprechend bei der Anlage des Vorgangs festgehalten werden, ob der Vorgang durch einen Katalogimport erzeugt wurde, um dann die relevanten Daten in der Datenbank zu hinterlegen.

In beiden Szenarien muss es definitiv möglich sein, die hinterlegte Information noch zu ändern, da sich z.B. PPNs von Vorgängen auch nach der Anlage in Kitodo ändern können. Dementsprechend müsste es zweitens auch mögliich sein, die Information nachträglich in Vorgängen zu hinterlegen, in denen sie fehlt.

Daher nochmal die Rückfrage an @solth und @oliver-stoehr, an welchem Ort (Ruleset, Datenbank) sich unter Berücksichtigung der obigen Erwägungen und der technischen Möglichkeiten sich die für den Reimport notwendigen Daten am sinnvollsten ablegen lassen.

**Reply by henning-gerhardt** [reply]
Ein paar Fragen von mir:
- Wurden sich auch Gedanken gemacht, wie sich die zusätzliche Speicherung dieser Information sich bei Bestandsdaten verhalten soll? Einige hundert tausend Datensätze von Hand nach zu tragen, kann ja keine Lösung sein oder?
- Ist vorgesehen bzw. wurde bedacht, dass sich Import-Konfigurationen auch ändern und abgelegte Informationen sich dann zwangsweise auch ändern müssen bzw. angepasst werden müssen. Wie soll dies in der Anwendung durchgeführt werden (auch wieder im Blick auf große Datenmengen).
- Wie soll sich der Re-Import verhalten, wenn die Daten aus mehreren Katalogen kommen?

**Reply by BartChris** [reply]
WIr haben über Massen-Reimports nachgedacht, uns aber aktuell auf den Re-Import einzelner Vorgänge konzentriert, um zumindest letzteres vollständig zu spezifizieren. In diesem Fall ist das Nachtragen von evtl. fehlender Informationen ein kleineres Problem.
Ein Re-Import aus mehreren Katalogen wäre eine Erweiterung der hier beschriebenen Szenarien, die wir - soweit ich übersehe - noch nicht durchdacht haben, es sollten aber grundsätzlich zumindest dieselben Regeln gelten, wenn ein zweiter Katalog konkurrierende Informationen zu bestimmten Feldwerten liefert.

Es wäre gut, wenn die Massenverarbeitunng von Re-Import-Vorhaben direkt auf den oben angestellten Überlegungen aufsetzen könnte.

Was die Ergänzung von Bestandsdaten angeht so könnte ich mir vorstellen, dass ein funktionaler Key (z.B. recordIdentifier) genutzt werden kann (ein entsprechendes Feld müsste in den Bestandstdaten natürlich enthalten sein...), um die für den Re-Import notwendigen Informationen (zumindest den Katalogschlüssel betreffend) batchweise nachzutragen. 
Wenn die Import-Konfiguration per Datenbank-Identifier referenziert würde, sollte ein Re-Import auf der zum Zeitpunkt gültigen Import-Konfiguration basieren. Falls grundsätzlichere Änderungen vorgenommen wurden (Import-Konfiguration existiert nicht mehr), würde das natürlich nicht mehr funktionieren. In jedem Fall bringt die Möglichkeit zukünftiger Re-Importe zusätzliche Komplexität bei Änderungen von Import-Konfigurationen.

In jedem Fall wäre gut, wenn es Möglichkeiten gäbe, die hinterlegte Import-Konfiguration und Katalogschlüssel für eine größere Menge von Vorgängen zu überprüfen. Z.B. als direkte Funktion oder im Rahmen der Sichtung eines CSV-Exports von Daten.

**Reply by solth** [reply]
Die aktuelle Implementierung sieht tatsächlich das funktionale Metadatum `recordIdentifier` im Regelsatz für die Konfiguration des Katalog-Identifiers vor. Dieses wird derzeit (noch) nicht für den Re-Import von Metadaten verwendet, sondern nur, um beim hierarchischen Import zu testen, ob eine potentiell mitzuimportierende Überordnung eines direkt an einer Suchschnittstelle abgefragten Datensatzes bereits im Kitodo-System als Vorgang existiert und dessen erneuter Import damit übersprungen werden soll oder nicht. 

Wenn diesem Metadatum beim Import - z.B. für die Anzeige - ein Präfix vorangestellt wird, sollte das automatische Erkennen einer vorhandenen Überordnung eigentlich nicht mehr funktionieren, d.h. ein als `recordIdentifier` ausgezeichnetes Feld sollte immer den unveränderten Quell-Datenbank-Identifier enthalten, den man zum erneuten Abfragen einer Schnittstelle wiederverwenden kann, wenn man den gleichen Datensatz erneut abrufen möchte. Ein zusätzlicher, modifizierter Identifier sollte in einem separaten Metadaten-Feld stehen, um die Funktionalität nicht zu beeinträchtigen.

Eine Möglichkeit, die Quelle eines importierten Metadatensatzes zu speichern, fehlt tatsächlich noch. Ein Grund dafür, auch diese Information in einem (neuen) funktionalen Metadatum unterzubringen, wäre aus meiner Sicht, dass die für den Re-Import benötigten Informationen an einem Ort zur Verfügung stehen würden und nicht zusammegensucht werden müssten. Wenn die Quelle stattdessen in der Datenbank gespeichert wird (z.B. anhand eines neuen Feldes "ImportConfigurationIId" an einem Vorgang), sollte man vermutlich auch den `recordIdentifier` in die Datenbank umziehen, um hier eine möglichst einheitliche Implementierung zu haben und die Wartbarkeit zu vereinfachen. Ob das Abspeichern der ImportConfigurationId in der Datenbank statt in einem Metadaten-Feld jedoch einen anderen Vorteil bringt, kann ich derzeit nicht sehen.

**Reply by BartChris** [reply]
> Wenn diesem Metadatum beim Import - z.B. für die Anzeige - ein Präfix vorangestellt wird, sollte das automatische Erkennen einer vorhandenen Überordnung eigentlich nicht mehr funktionieren, d.h. ein als `recordIdentifier` ausgezeichnetes Feld sollte immer den unveränderten Quell-Datenbank-Identifier enthalten, den man zum erneuten Abfragen einer Schnittstelle wiederverwenden kann, wenn man den gleichen Datensatz erneut abrufen möchte. Ein zusätzlicher, modifizierter Identifier sollte in einem separaten Metadaten-Feld stehen, um die Funktionalität nicht zu beeinträchtigen.
> 

Wenn bei der entsprechenden Überordnung dasselbe gemacht wird, d.h. wenn immer eine PPN vorangestellt wird, funktioniert die automatische Verknüpfung zur Überordnung vermutlich trotzdem. Insofern war vermutlich nicht intendiert, dass im `recordIdentifier` etwas anderes steht als der exakte Katalogschlüssel, allerdings wird dies softwareseitig aktuell nicht erzwungen. Und  die automatische Verknüpfung zur Überordnung kann dann trotzdem funktionieren, wenn es immer (also auch für die Überordnungen und die Verweise auf die Überordnungen) gemacht wird.

**Reply by solth** [reply]
> Wenn bei der entsprechenden Überordnung dasselbe gemacht wird, d.h. wenn immer eine PPN vorangestellt wird, funktioniert die automatische Verknüpfung zur Überordnung vermutlich trotzdem. Insofern war vermutlich nicht intendiert, dass im `recordIdentifier` etwas anderes steht als der exakte Katalogschlüssel, allerdings wird dies softwareseitig aktuell nicht erzwungen. Und die automatische Verknüpfung zur Überordnung kann dann trotzdem funktionieren, wenn es immer (also auch für die Überordnungen und die Verweise auf die Überordnungen) gemacht wird.

Richtig, wenn das funktionale Metadatum, das als `higherlevelidentifier` konfiguriert wurde (und das die ID der Überordnung enthalten soll) auf die gleiche Art modifiziert wurde wie der `recordIdentifier` selbst, funktioniert auch die automatische Verknüpfung mit der Überordnung (da der `higherlevelidentifier` der Unterordnung hierbei mit dem `recordIdentifier` der Überordnung verglichen wird). Auf die Idee, _beide_ Werte gleichmäßig zu ändern und damit weiterhin kompatibel zu halten, bin ich noch nicht gekommen. 

Allerdings wird dies vermutlich beim Thema "Re-Import" tatsächlich Probleme verursachen, da die Suchschnittstelle nichts über den zusätzlichen Präfix weiß und Kitodo bisher auch keine Funktionalität enthält, einen solchen (optionalen) Präfix explizit (e.g. außerhalb der Import-XSLT-Datei) zu definieren und damit vor erneuter Abfrage der Suchschnittstelle automatisch zu entfernen.

**Reply by BartChris** [reply]
Evtl. können @subhhwendt oder @apiller kurz erläutern mit welcher Intention die Anreicherung der Katalogschlüssel mit einem Präfix erfolgt. Ich glaube teilweise sind zu diesem Zweck auch schon Doppelfelder im Einsatz um etwa die ursprüngliche PPN und die PPN mit Präfix beide abzubilden.

**Reply by apiller** [reply]
Wenn ich das richtig in Erinnerung habe ist das bei @subhhwendt im Einsatz, um eine Verwechselung mit Identifiern aus anderen Katalogen zu verhindern. Wir haben zumindest in Kitodo 3 keine Präfixe im Einsatz. In der 2er Version haben wir damit sichergestellt, dass unser Mandant eine bei uns bereits angelegte PPN ohne Probleme nochmal anlegen kann. Problem hierbei war, dass die PPN als ID auch für Präsentation und Verknüpfung zu übergeordneten Stufen verwendet wird. Der recordIdentifier ist aber bei uns immer die aktuelle PPN ohne Prä- oder Suffixe.

**Reply by subhhwendt** [reply]
Wir sind 2010ff an der Seite von Intranda im Schatten der Göttinger Installation mit dem Präfix "PPN" gestartet - ohne das bis in alle Tiefen zu durchdenken. Ich meine auch die SBB ist so unterwegs. 
Das Konfigurationsgewusel fing - wie schon von Arved beschrieben- mit Kitodo3 an, wo der Vorgang der Überordnung zuverlässig gefunden werden musste. So ist ab Version 3.6 in unserem Regelsatz zu finden - vorher war ein Teilaspekt in der Importkonfiguration untergebracht ("Metadatum für Katalog-ID"):

<key id="CatalogIDDigitalInternal" use="recordIdentifier"> =>KatalogIdentifier ohne Vorlauf PPN
<key id="CatalogIDDigitalAnchorInternal" use="higherlevelIdentifier"> =>KatalogIdentifier der ÜO in meta.xml eines Bandes, der dann auf die CatalogIDDigitalInternal bei der ÜO matcht.

**Reply by andre-hohmann** [reply]
Ich wurde darauf hingewiesen, dass in der SLUB Dresden ebenfalls Vorgänge in Kitodo.Production verwaltet werden, in denen `recordIdentifier` nicht dem Identifier entspricht, mit dem die Schnittstelle durchsucht wird. Statt dessen wird der recordIdentifier aus zwei Werten zusammengesetzt. 

Deshalb  unterstütze ich mittlerweile zur Kennzeichnung des Identifikators für den Re-Import die Erstellung eines neuen Attributes für den Regelsatz, wie zum Beispiel `importMetadata`. 

Das scheint mir am nachhaltigsten zu sein. Im schlimmsten Fall werden einem Metadatum zwei Attribute zugewiesen. 
`<key id="CatalogIDDigital" use="recordIdentifier" use="importMetadata">`
Da sich das Metadatum vermutlich selten ändern wird, ist der Aufwand aus meiner Sicht überschaubar. Die Flexibilität für mögliche zukünftige Anwender oder Anwendungsfälle ist jedoch sehr hoch.

**Comment by subhhwendt** [comment]
da ist was vom Editor gefressen worden - sorry!
CatalogIdDigitalInternal mit use="recordIdentifier" =>KatalogIdentifier ohne Vorlauf PPN
CatalogIdDigitalAnchorInternal mit use="higherlevelIdentifier"=>KatalogIdentifier der ÜO in meta.xml eines Bandes, der dann auf die CatalogIDDigitalInternal bei der ÜO matcht.

**Comment by andre-hohmann** [comment]
Hier noch Antworten auf gestellte Fragen

> Wurden sich auch Gedanken gemacht, wie sich die zusätzliche Speicherung dieser Information sich bei Bestandsdaten verhalten soll? Einige hundert tausend Datensätze von Hand nach zu tragen, kann ja keine Lösung sein oder?

Ja, das wurde bedacht und intern diskutiert. Wenn neue Elemente oder Funktionen implementiert werden, soll die notwendige maschinelle Anpassung der bestehenden Vorgänge ermöglicht werden. Dies betrifft vor allem die mögliche Verknüpfung mit der jeweiligen Importkonfiguration. 
Eine manuelle Anpassung der vorhandenen Vorgänge, um einen Re-Import zu ermöglichen muss unbedingt vermieden werden. 

> Ist vorgesehen bzw. wurde bedacht, dass sich Import-Konfigurationen auch ändern und abgelegte Informationen sich dann zwangsweise auch ändern müssen bzw. angepasst werden müssen. Wie soll dies in der Anwendung durchgeführt werden (auch wieder im Blick auf große Datenmengen).

Nein, das wurde noch nicht bedacht. Das ist vermutlich auch davon abhängig, wie die Funktion implementiert wird. Im besten Fall wird jeweils die Import-Konfiguration entsprechend angepasst, um eine Anpassung der Vorgangs-Eigenschaften (ID der Import-Konfiguration, ...) zu vermeiden. 
Diesbezüglich müssten Anwendungsfälle beschrieben werden. 
Unter Umständen kann die Lösung zur vorigen Fragen angewendet/angepasst werden. 

> Wie soll sich der Re-Import verhalten, wenn die Daten aus mehreren Katalogen kommen?

Das wurde noch nicht ausführlich diskutiert, da bisher in der Regel nur die Metadaten aus einem Katalog importiert werden.

**Comment by andre-hohmann** [comment]
Es wurde eine zweite Version erstellt, in der weitere Erkenntnisse und Ergebnisse aus dem weitere [Community Board](https://github.com/kitodo/kitodo-production/wiki/Community-Board-(Kitodo.Production)) zusammengeführt wurden: 
- https://github.com/kitodo/kitodo-production/discussions/5904

Hinsichtlich der Übersichtlichkeit wird diese Diskussion geschlossen.
